subprojects/common/core/src/main/java/hu/bme/mit/theta/core/utils/FpTestUtils.java

```diff
@@ -0,0 +1,181 @@
+package hu.bme.mit.theta.core.utils;
```

Comment:
It is used both by FpTypeTest and by Z3SolverFPTest, and therefore it is only accessible by both this way (if it was in any of the test packages, only one subproject could access to AFAIK)

Reply:
Gradle can be convinced otherwise, if it is not the default setting.

If I am not mistaken, adding something like this line could help:
https://github.com/ftsrg/theta/blob/master/subprojects/cfa/cfa-analysis/build.gradle.kts#L10

Reply:
Thanks, I'll see what I can do about it

Reply:
We arrived at the conclusion that it is not worth the hassle, because Gradle does not want to cooperate. So for now this class shall remain in its original place.

Reply:
I've thought I've came across this issue some time ago, but it is a more complex one... I was mistaken: testCompile/testImplementation does not do this (it is created for importing testing utilities, such as JUnit)...

A clear solution would be to create a core-test-utils subproject, which exports the test thingies.
This would blow up easily (I mean combinatorically: for every subproject for one or two test files, a whole subproject...?)...

There could also be some Gradle magic, with which one can import the `test` from another project, but I do not know how to achieve this

Reply:
With this I could successfully move the class out of the _main_ source-set. Thanks for the help!